etl/planning_data/load_into_database.py

```diff
@@ -0,0 +1,122 @@
+    limit = 400
+    if len(description) > limit:
+        description = ""
+        for entry in original_description.split():
+            extended = description
+            if extended != "":
+                extended += " "
+            extended += entry
+            if len(extended) <= limit:
+                description = extended
+        if description == "":
+                description = description[0:limit]
+        description += "... <i>(show more)</i>"
+    return description
+
+def main():
+    connection = get_connection()
+    with connection.cursor() as cur:
+        cur.execute("TRUNCATE planning_data")
+    with open(filepath(), 'r') as content_file:
+        data = json.load(content_file)
+        if data['rawResponse']['timed_out']:
+            raise Exception("query getting livestream data has failed")
+        if data['is_partial']:
+            raise Exception("query getting livestream data has failed")
+        if data['is_running']:
+            raise Exception("query getting livestream data has failed")
+        for entry in data['rawResponse']['hits']['hits']:
+            description = shorten_description(entry['_source']['description'])
+            application_id = entry['_source']['id']
+            decision_date = parse_date_string_into_datestring(entry['_source']['decision_date'])
+            last_synced_date = parse_date_string_into_datestring(entry['_source']['last_synced'])
+            uprn = entry['_source']['uprn']
+            status = entry['_source']['status']
+            if status in ["No Objection to Proposal (OBS only)", "Not Required", None, "Lapsed", "Unknown", "SECS", "Comment Issued"]:
+                continue
+            if status in []:
+                opts = jsbeautifier.default_options()
+                opts.indent_size = 2
+                print(jsbeautifier.beautify(json.dumps(entry), opts))            
+                continue
+            if status == "Refused":
+                status = "Rejected"
+            if status == "Appeal Received":
+                status = "Appeal In Progress"
+            if (status not in ["Approved", "Rejected", "Appeal In Progress", "Withdrawn", ]):
+                raise Exception("Unexpected status " + status)
+            description = entry['_source']['description'].strip()
+            if uprn == None:
+                continue
+            entry = {
+                "description": description,
+                "decision_date": decision_date,
+                "last_synced_date": last_synced_date,
+                "application_id": application_id,
+                "application_url": entry['_source']['url_planning_app'],
+                "uprn": uprn,
+                "status": status,
+                "data_source": "The Planning London DataHub Greater London Authority",
+                "data_source_link": "https://data.london.gov.uk/dataset/planning-london-datahub?_gl=1%2aprwpc%2a_ga%2aMzQyOTg0MjcxLjE2NTk0NDA4NTM", # TODO test
```

Comment:
Just checking were you intending to change something here? The link shows the map for Brent in London only, but I don't know if this is just the front end default selection and irrelevant? Also just to mention there are a couple of other TODOs scattered in the code